scale/job/models.py

```diff
@@ -1052,6 +1052,10 @@ def schedule_job_executions(self, framework_id, job_executions, workspaces):
             job_exe.started = started
             job_exe.node = node
             job_exe.configure_docker_params(framework_id, workspaces)
+            
+            for key, value in job_exe.job.job_type.docker_params:
+                job_exe.configuration['job_task']['docker_params'].append({'flag': key, 'value': value})
+                
```

Comment:
1. Please move this to JobExecution.configure_docker_params(). This is the location where Docker parameters are set up for the job execution.
2. Please use configuration.add_job_task_docker_param(DockerParam(key, value)) instead of directly hitting the internals of the configuration dict.
3. Please add a conditional to ensure that docker_params is not None. It's possible for the field to be set to None.